### PR TITLE
expose OpenAPI specifications file

### LINKED ISSUE #95: Expose openapi.json
According to https://platform-docs.cloud.paas.psi.redhat.com/rest/requirements.html

```
For consistency and to support automated testing of APIs, all applications MUST serve the current OpenAPI specification in JSON format on $PATH_PREFIX/$APP/v$VERSION/openapi.json.
```

### PR DESCRIPTION
**Pull request by Bee-lee**
# Description
Exposes OpenAPI specs file specified in config.toml as per the [requirements](https://platform-docs.cloud.paas.psi.redhat.com/rest/requirements.html)

Fixes #95 

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

## Testing steps
`go test`, `test.sh`, app won't run without it the file loaded.

### CODE REVIEW

File: configuration.go [hunk 1]
```diff
@@ -149,6 +149,18 @@ func loadOrganizationWhitelist() mapset.Set {
 	return whitelist
 }
 
+func getAPISpecFile() (string, error) {
+	specFile := serverCfg.GetString("api_spec_file")
+
+	fileInfo, err := os.Stat(specFile)
+	if os.IsNotExist(err) {
```
Comment (tisnik) [inline configuration.go]: prob. might be good to add a test & !os.IsDir() or & os.IsRegular()

File: server/server.go [hunk 2]
```diff
@@ -193,6 +194,12 @@ func (server HTTPServer) readReportForCluster(writer http.ResponseWriter, reques
 	}
 }
 
+// serveAPISpecFile serves an OpenAPI specifications file specified in config file
+func (server HTTPServer) serveAPISpecFile(writer http.ResponseWriter, request *http.Request) {
+	absPath, _ := filepath.Abs(server.Config.APISpecFile)
```
Comment (tisnik) [inline server/server.go]: well the file existence was tested before, but pls. add err checks there as well

Comment (Bee-lee) [reply server/server.go]: fixed

File: configuration.go (hunk 1 above)
Comment (Bee-lee) [reply configuration.go]: added, also added another check in test.sh

**Comment by Bee-lee** [comment]
About the decrease of test coverage - I wanted to add unit tests for it, but I had some weird trouble because I couldn't use the regular configuration functions (they're in main). I will add unit tests for it once https://github.com/RedHatInsights/insights-results-aggregator/issues/126 is done

**Comment by Bee-lee** [comment]
@tisnik :D thanks

**Pull request merged**